### DISCUSSION TITLE
check this one out

### DISCUSSION THREAD

**Discussion by adrianpuiu** [seruva19/kubin#36] [General]
https://research.nvidia.com/labs/par/Perfusion/ , code coming out soon from nvidia

**Comment by seruva19** [comment]
Wow, that's cool stuff. Even though it is claimed to require 27 Gb of VRAM for training 😑

**Reply by adrianpuiu** [reply]
for training i'm covered. i got 48Gigs of Vram .. but having 100 kb models , hah , that will save my hard drives space big time :))


+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 530.41.03              Driver Version: 530.41.03    CUDA Version: 12.1     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                  Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf            Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA GeForce RTX 4090         Off| 00000000:01:00.0 Off |                  N/A |
|  0%   40C    P8               41W / 450W|      8MiB / 24576MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
|   1  NVIDIA GeForce RTX 4090         Off| 00000000:02:00.0  On |                  Off |
|  0%   42C    P5               35W / 450W|   8808MiB / 24564MiB |     22%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
